app/src/main/java/me/quadphase/qpdex/databaseAccess/ExternalDbOpenHelper.java

```diff
@@ -43,8 +43,8 @@ public ExternalDbOpenHelper(Context context, String databaseName) {
     //This piece of code will create a database if it’s not yet created
     public void createDataBase() {
         boolean dbExist = checkDataBase();
-        // TODO: During scraper development, change to "if (true)". After development, change back to "if(!dbExist)"
-        if (!dbExist) {
+        // NOTE: During DB development, change to "if (true)". For release, change back to "if(!dbExist)"
+        if (true) {
```

Comment:
I had changed this to !dbExists because then it held the caught pokemon info. I guess we're still in dev so it doesn't matter.